funnel/templates/project_layout.html.jinja2

```diff
@@ -358,7 +338,7 @@
             </div>
           {%- elif sponsorship.member.description %}
             <div class="mui--text-body2 mui--text-light margin-bottom">
-              {{ sponsorship.member.description|preview(min=200, max=300) }}
+              {{ sponsorship.member.description.html|preview(min=200, max=300) }}
```

Comment:
This has been fixed in Baseframe. Using `.html` for the `|preview` filter is no longer necessary.